<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>main_window</class>
 <widget class="QWidget" name="main_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>408</width>
    <height>213</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gmail Resender</string>
  </property>
  <widget class="QLabel" name="label_main">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>20</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Gmail Resender</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_start">
   <property name="geometry">
    <rect>
     <x>168</x>
     <y>166</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Start</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="text_edit_sender">
   <property name="geometry">
    <rect>
     <x>91</x>
     <y>61</y>
     <width>299</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="text_edit_recipient">
   <property name="geometry">
    <rect>
     <x>91</x>
     <y>125</y>
     <width>299</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="text_edit_password">
   <property name="geometry">
    <rect>
     <x>91</x>
     <y>93</y>
     <width>299</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_password">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>96</y>
     <width>67</width>
     <height>19</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Password</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_recepient">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>130</y>
     <width>64</width>
     <height>19</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Recipient</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_sender">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>62</y>
     <width>49</width>
     <height>19</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Sender</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
